python/requirements.txt

```diff
@@ -1,6 +1,5 @@
-black
-cmake>=3.14
-isort>5
-pylint
-pytest>=5,<6
-wheel
+lark-parser==0.12.0
```

Comment:
Should `lark-parser` actually be a requirement here? Isn't that a requirement only for `quantum-xir` (making it only indirectly a requirement for Jet)?

Reply:
Nope, thanks for catching this! 🥅